### DISCUSSION TITLE
Charts and graphs

### DISCUSSION THREAD

**Discussion by bgmort** [nushell/nushell#11843] [Ideas]
Recently, I've discovered the `histogram` command, and found it very helpful in slicing and analyzing data. I'd love to see support for more data visualization in nushell as well. I don't know if it would be best as a core feature, plugin, or community contribution. But, to be able to easily construct and view plots of data over time, such as with line or bar charts, would be a really useful and valuable feature.

**Comment by fdncred** [comment]
If you look in our awesome-nu repo, there is a plugin that does other types of console charts [nu_plugin_plot](https://github.com/Euphrasiologist/nu_plugin_plot). I'm not sure if it's been kept up to date though. Someone was also working on a gui chart application that launched a web browser or something like that.

**Comment by tino376dev** [comment]
I also think the histogram chart is really helpful. That said, the primitive bar chart implied by the histogram would also be nice to expose.
